### DISCUSSION TITLE
interaction.options.getSubcommand is not a function

### DISCUSSION THREAD

**Discussion by Tom-ne** [discordjs/discord.js#7158] [Q&A] [answered]
# The problem
I am making a bot that uses slash commands. 
I made a command that worked on a previous version of the package that used sub-commands but when I try to use the command now
I get an error ```TypeError: interaction.options.getSubcommand is not a function``` I checked the documentation and console.log the interaction I also tried to reinstall discord.js but I can't find a way to make it work.
Can you help me?

**Comment by VoltrexKeyva** [comment, marked as answer]
Your `interaction` variable/parameter is possibly not an instance of the `CommandInteraction` class, which is the result of either passing parameters in the incorrect order to your command handler in a way (If you're using a command handler of course), or you're passing the incorrect value to that variable if it is one.

Better to ask for help about that and similar issues related to discord.js by joining [discord.js' official Discord support server](https://discord.gg/djs).

**Reply by Tom-ne** [reply]
Thanks for the help. I found the problem

**Reply by Kerem24** [reply]
How did you fix it? Did you create a param or something else?
